jekyll/_ccie/backup.md

```diff
@@ -1,49 +1,51 @@
 ---
 layout: enterprise
 section: enterprise
-title: "Backing up CircleCI Enterprise"
+title: "Backing up CircleCI Data"
 category: [advanced-config]
 order: 8
 description: "How to regularly back up CircleCI Enterprise"
 ---
 
-This document will describe how to back up your CircleCI Enterprise application so that you can recover from accidental or unexpected loss of the VM containing all of your persistent CircleCI data (the services box) in the following sections:
+This document describes how to back up your CircleCI Enterprise application so that you can recover from accidental or unexpected loss of the Services machine VM that contains CircleCI data in the following sections:
```

Comment:
Formally speaking the data lives on a volume attached to the VM. We could probably say something more like "accidental or unexpected loss of CircleCI data attached to the Services machine." (style note: I think "machine" is probably OK by itself - feels bit redundant to say "machine VM"